### ISSUE TITLE
Wrong information message in ethereum-bridge on ubuntu

### ISSUE THREAD

**Issue by robertmagier** [oraclize/ethereum-bridge#80]
After starting ethereum bridge on Ubuntu it will display message:
`OAR = OraclizeAddrResolverI(0x1160BceeFDfc731C96244d96de946B9769d16E16);`

This is wrong because it should be:
`OAR = OracleAddrResolverI(0x1160BceeFDfc731C96244d96de946B9769d16E16);`

Version:
`INFO you are running ethereum-bridge - version: 0.6.2`

**Comment by vm06007** [comment]
yes I got the same, but then it does allow me to deploy a contract once that part is added into the constructor, however, calling provable_newRandomDSQuery() fails for me locally, any idea?
